### DISCUSSION TITLE
How to change the color of connected account informations without missing out with w3m-router?!

### DISCUSSION THREAD

**Discussion by L1xus** [WalletConnect/walletconnect-monorepo#4309] [Developer Support] [answered]
Hi, I'm trying to change the theme of my app and wanna add specific colors to specific parts!
I tried to override the default style of the module with css but failed :/
I'm using this style now in createWeb3Model 
`themeVariables: {
    '--w3m-accent': '#fe7e01',
    '--w3m-border-radius-master': '1px',
    '--w3m-color-mix': "#333333",
    '--w3m-color-mix-strength': 100 
  } `
  but I get this problem!
  
![20240309_02h02m03s_grim](https://github.com/WalletConnect/walletconnect-monorepo/assets/117644284/c6db62ad-d01c-4e3a-91d7-30df856b9bb8)

how can I solve this plz!

**Comment by glitch-txs** [comment, marked as answer]
Hi thanks for reporting! Could you open this issue [here](https://github.com/WalletConnect/web3modal/issues)?

**Reply by L1xus** [reply]
yes sure!
